teams/compiler-contributors.toml

```diff
@@ -57,3 +57,12 @@ extra-teams = ["compiler"]
 name = "Compiler team contributors"
 repo = "http://github.com/rust-lang/compiler-team"
 description = "Contributing to the Rust compiler on a regular basis"
+
+[[zulip-groups]]
+name = "T-compiler"
+
+[[zulip-groups]]
+name = "T-compiler/meeting"
+
+[[zulip-groups]]
+name = "T-compiler/contrib-private"
```

Comment:
How does group membership translate to stream membership? I thought those were different concepts in Zulip.

Reply:
oh 🤦 I just saw the matching names and conveniently overlooked the `group` part of `zulip-groups`.

oof. zulip will just add individual users from a team to a stream if you add a team. So we'll need to manage this completely on our end.